spec/factories/course_enrichments.rb

```diff
@@ -98,4 +98,23 @@
     status { :draft }
     last_published_timestamp_utc { 5.days.ago }
   end
+
+  trait :invalid_content do
```

Comment:
Prehaps this should be more explicit i.e. `trait :invalid_enrichment do`

Reply:
let(:invalid_enrichment) { create(:course_enrichment, `:with_invalid_content`) }

how about `:with_invalid_content`?